Public/Sdk/SelfHost/BuildXL/BuildXLSdk.dsc

```diff
@@ -286,6 +284,7 @@ export function executable(args: Arguments): Managed.Assembly {
         ],
         tools: {
             csc: {
+                platform: <"x64">"x64",

```

Comment:
Is this some copy paste mistake?

Reply:
A potential bug in type checking forces Danny to do the casting.